client/app/components/auth/LogInButton.tsx

```diff
@@ -1,32 +1,12 @@
 import React from "react";
-import { StyleSheet, Text, TouchableOpacity, Dimensions } from "react-native";
 
 import { LogInButtonProps } from "../../types/Props";
+import LargeTextButton from "./LargeTextButton"
 
 const LogInButton: React.FC<LogInButtonProps> = ({ onPress }) => {
   return (
```

Comment:
Instead of replacing the LogInButton with a single large text button component, just delete the loginbutton and signupbutton files and modify where they were previously used to now use the new button.
With all 3 buttons existing at the same time, there is a bit of redundant code.

Reply:
actually the login and signup button was getting exported from authbutton to login and signup screen instead of the login and signup button component so deleted the login and signup components and removed the login and signup button code and style from  authbutton and created the largeTextButton with the style and also removed the login and signup prop